src/yunohost/domain.py

```diff
@@ -220,17 +220,20 @@ def domain_remove(auth, domain, force=False):
     msignals.display(m18n.n('domain_deleted'), 'success')
 
 
-def domain_dns_conf(domain):
+def domain_dns_conf(domain, time_to_live):
     """
     Generate DNS configuration for a domain
 
     Keyword argument:
         domain -- Domain name
+        time to live -- Time to live
```

Comment:
We should be more consistent with arguments & option name; 
In the actionmap we use `ttl`, function's argument is `time_to_live`, and in function comment it's `time to live` without underscores. Maybe it's just aesthetic, but it could lead to cumbersome documentation.